src/Common/TypeExtensions.cs

```diff
@@ -38,7 +39,14 @@ public static Type GetType(this Type interfaceType, string typeString)
                       typeString,
                       (name) =>
                       {
-                          return AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(z => !string.IsNullOrWhiteSpace(z.FullName) && z.FullName.StartsWith(name.FullName));
+                          var assembly = AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(z => !string.IsNullOrWhiteSpace(z.FullName) && z.FullName.StartsWith(name.FullName));
+
+                          if (assembly is null)
+                          {
+                              assembly = Assembly.LoadFile($"{AppDomain.CurrentDomain.BaseDirectory}{name.FullName}.dll");
```

Comment:
src/Common/TypeExtensions.cs#L46

Replace this call to 'Assembly.LoadFile' with 'Assembly.Load'.

Reply:
Might be worth doing this change